enterprise/signals/selections.py

```diff
@@ -119,6 +119,25 @@ def nanograv_backends(backend_flags):
     return {flagval: backend_flags == flagval for flagval in flagvals}
 
 
+def custom_backends(cb):
+    def backends(backend_flags):
+        """Selection function to split by custom backend flags only.
+        cb : list of str of the backends
+        use None to recover by_backend
+        use ["ASP", "GASP", "GUPPI", "PUPPI"] to recover nanograv_backends
+        """
+        nonlocal cb
+        flagvals = np.unique(backend_flags)
+        if cb is not None:
+            cb = list(np.atleast_1d(cb))
+            flagvals = filter(lambda x: any(map(lambda y: y in x, cb)), flagvals)
```

Comment:
Is this list comprehension more readable?
```suggestion
            flagvals = [flag for flag in flagvals if any(this_cb in flag for this_cb in cb]
```

Reply:
I copied that line from `nanograv_backend`, so whatever looks nicer to people and passes the formatting checks, is good to me.